### PR TITLE
NMS-14537: Add option to disable IO collection

### PR DESCRIPTION
**Pull request by amay94**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14537

### CODE REVIEW

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 1]
```diff
@@ -234,4 +246,4 @@ OPTIONS
                 Whether to persist config or not
 ----
 
-NOTE: Above karaf commands won't persist config by default, use -p option to persist config in {page-component-title} Database.
\ No newline at end of file
+NOTE: Above karaf commands won't persist config by default, use -p option to persist config in {page-component-title} Database.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/configuration.adoc:249]:
```suggestion
NOTE: The previous Karaf commands will not persist configuration by default.
Use the -p option to persist configuration in the {page-component-title} database.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor changes to the docs.

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 2]
```diff
@@ -83,6 +83,18 @@ config:property-set tftpServerIPv6Address ::1
 config:update
 ----
 
+== SSH Session I/O Collection
+
+By default, the ssh scripting service collects all ssh session output for debugging purposes and saves it in the poller's status. 
+To disable this, you can configure the scripting service on both Horizon and Minion as below:
```
[the remark this answers is not in the document]
Comment (amay94) [reply docs/modules/operation/pages/device-config-backup/configuration.adoc]: You need to disable it on all minions, updated.

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 3]
```diff
@@ -83,6 +83,18 @@ config:property-set tftpServerIPv6Address ::1
 config:update
 ----
 
+== SSH Session I/O Collection
+
+By default, the ssh scripting service collects all ssh session output for debugging purposes and saves it in the poller's status.
```
[the remark this answers is not in the document]
Comment (amay94) [reply docs/modules/operation/pages/device-config-backup/configuration.adoc]: It's stored in the poller's status so the karaf shell has access to it and can print it if needed. But everything that has access to the poller's status has access to it as well.

[the remark this answers is not in the document]
Comment (amay94) [reply docs/modules/operation/pages/device-config-backup/configuration.adoc]: I changed 'saves' to 'stores', as 'save' could imply persisting in a database.

**Pull request merged**